cypress/e2e/about.cy.ts

```diff
@@ -7,73 +7,73 @@ describe('About page', () => {
             cy.visit('/about')
         })
 
-        it('shows the desktop nav', () => {
-            cy.get('[data-testid="desktop-nav"]').should('exist')
-        });
+        it('shows the desktop top nav', () => {
+            cy.get('[data-testid="landscape-searchbar"]').should('exist')
```

Comment:
Since the element will always be present in the DOM with these changes, the test should confirm that it's not visible.
```suggestion
            cy.get('[data-testid="landscape-searchbar"]').should("not.be.visible")
```

Reply:
good call